### PR TITLE
ntfs-3g : update to latest stable version

### PR DESCRIPTION
**Pull request by hbl0307106015**
Maintainer: Ted Hess
Compile tested: ramips, Y1, OpenWRT/LEDE latest version
Run tested: ramips, Y1, OpenWRT/LEDE latest version, tests done

The new stable version of ntfs-3g is release

Changes to NTFS-3G:

Delegated processing of special reparse points to external plugins
Allowed kernel cacheing by lowntfs-3g when not using Posix ACLs
Enabled fallback to read-only mount when the volume is hibernated
Made a full check for whether an extended attribute is allowed
Moved secaudit and usermap to ntfsprogs (now ntfssecaudit and ntfsusermap)
Enabled encoding broken UTF-16 into broken UTF-8
Autoconfigured selecting <sys/sysmacros.h> vs <sys/mkdev>
Allowed using the full library API on systems without extended attributes support
Fixed DISABLE_PLUGINS as the condition for not using plugins
Corrected validation of multi sector transfer protected records
Denied creating/removing files from $Extend
Returned the size of locale encoded target as the size of symlinks

please visit below for more details:
http://www.tuxera.com/community/release-history/

### CODE REVIEW

File: utils/ntfs-3g/Makefile [hunk 1]
```diff
@@ -215,8 +215,8 @@ endef
 
 define Package/ntfs-3g-utils/install
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ntfs-3g.secaudit $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ntfs-3g.usermap $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/man/man8/ntfssecaudit.8 $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/man/man8/ntfsusermap.8 $(1)/usr/bin
```
Comment (obsy) [inline utils/ntfs-3g/Makefile]: Why man files is installed in /usr/bin?

Comment (thess) [inline utils/ntfs-3g/Makefile]: These 2 utilities have been merged into ntfsprogs. See the 5th item in the change list above. This change is in error.

**Comment by thess** [comment]
@hbl0307106015 - Thanks for your submission. I have re-worked the Makefile a bit and combined the extra utils into a single ntfs-3g-utils package. New commit here: 092b4d0ae830b99f69ae0865560571609a11650f

**Comment by hbl0307106015** [comment]
@thess  @hnyman Sorry for my mistake, I ought to have modified it more early !

**Pull request closed without merging**